submissions/xT0PERx/Memory pair game/transitions.js

```diff
@@ -0,0 +1,94 @@
+const images = [
+  'images/Alfa-romeo.jpeg',
+  'images/Ford.jpeg',
+  'images/Mazda.jpeg',
+  'images/Nissan.jpeg',
+  'images/Peugeot.jpeg',
+];
+
+const cardDeck = [...images, ...images];
+const duration = 600;
+let winCount = 0;
+let firstOpenCard = null;
+let isBlock = false;
+const pointsWon = images.length;
+const gameBoard = document.querySelector('.cards');
+
+function getSortList() {
```

Comment:
Usually method's name starting with `get-` implies returning something. This method does not return anything, instead it modifies data in place. I would suggest renaming it to `sortList`. Alternatively you can keep the name and return list from the function.